diff -rupN --no-dereference libwebp-1.3.0/CMakeLists.txt libwebp-1.3.0-new/CMakeLists.txt
--- libwebp-1.3.0/CMakeLists.txt	2023-01-13 13:06:13.664139980 +0100
+++ libwebp-1.3.0-new/CMakeLists.txt	2023-01-13 13:06:13.669140015 +0100
@@ -101,11 +101,11 @@ endif()
 include(cmake/deps.cmake)
 include(GNUInstallDirs)
 
-if(BUILD_SHARED_LIBS AND NOT CMAKE_INSTALL_RPATH)
+#if(BUILD_SHARED_LIBS AND NOT CMAKE_INSTALL_RPATH)
   # Set the rpath to match autoconf/libtool behavior. Note this must be set
   # before target creation.
-  set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}")
-endif()
+  # set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}")
+#endif()
 
 # ##############################################################################
 # Options.
